src/api/routes/meetings.ts

```diff
@@ -113,9 +114,49 @@ const decline = procedure
     商量解决方案。`, baseUrl);
 });
 
+/**
+ * List all meeting slots for admin management.
+ * Only MentorshipManager can access this.
+ */
+const listSlots = procedure
+  .use(authUser("MentorshipManager"))
+  .query(async () => {
+    return await db.MeetingSlot.findAll({
+      attributes: meetingSlotAttributes,
+      order: [["updatedAt", "DESC"]],
+    });
+  });
+
+/**
+ * Update a meeting slot's information.
+ * Only MentorshipManager can modify meeting slots.
+ */
+const updateSlot = procedure
```

Comment:
I just thought of one more requirement: support adding new meeting slots. so please add a route `createMeetingSlot`.